src/elements/list-box.svelte

```diff
@@ -112,11 +137,14 @@ const handleMoveClick = (target: ListBoxSide) => {
       {#if options.left.length > 0 }
        {#each options.left as option}
           <button
-            class={cx('flex items-center px-2 py', { 'bg-focus/highlight': option.selected })}
+            class={cx('flex items-center px-2 py text-sm', { 'bg-focus/highlight': option.selected })}
             on:click={() => handleOptionClick(option, LEFT)}
           > 
             <input type='checkbox' checked={option.selected} disabled={isDisabled}/>       
-            <span class="text-sm px-4">{ option.value }</span>
+            <span class="px-4">{ option.value }</span>
+            {#if displaySuffix && option.suffix}
```

Comment:
right now, if you put a super long title/suffix, it'll expand the box and squish the other one, although it won't occur much in current use cases, might be a nice thing to fix later. But personally would make the letters hidden at some point. Fine for now
<img width="1438" alt="Screen Shot 2023-02-15 at 2 02 46 PM" src="https://user-images.githubusercontent.com/29152328/219127923-7d036c25-d1cf-44e3-951c-5e9c856970a6.png">

Reply:
overflow-x?